src/config/urls.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Allows for runtime configuration. Internally, the runtime should import this
+ * module for various constants. We can use the AMP_CONFIG global to translate
+ * user-defined configurations to this module.
+ * @param {string} key
+ * @return {any}
+ */
+const env = (key) => (self.AMP_CONFIG ? self.AMP_CONFIG[key] : null);
+
+/**
+ * @param {T} value
+ * @return {T}
+ * @template {T}
+ */
+const pure = (value) => value;
```

Comment:
This should either be moved to a utility helper (and test that the callsite is referencing that helper), or we should use PURE comment directly.